python/python/glide/async_commands/transaction.py

```diff
@@ -57,7 +58,7 @@ def get(self, key: str):
         Command response:
             Optional[str]: If the key exists, returns the value of the key as a string. Otherwise, return None.
         """
-        self.append_command(RequestType.GetString, [key])
+        return self.append_command(RequestType.GetString, [key])
```

Comment:
What about adding a return type of these functions? what's the typing convention with function chaining?

Reply:
done